src/components/Chat/ChatBubble.tsx

```diff
@@ -93,7 +93,7 @@ class ChatMessage extends UIComponent<Extendable<IChatMessageProps>, any> {
     rest,
     styles,
     variables,
-  }: IRenderResultConfig<IChatMessageProps>) {
+  }: IRenderResultConfig<IChatBubbleProps>) {
```

Comment:
hmm.. if you take a look at the proposal in our internal ticket, the idea was that the `ChatBubble` would contain the body of a message + media and avatar would be part of `ChatItem`; is that achievable?

![image](https://user-images.githubusercontent.com/5442794/45876204-48469a80-bd9a-11e8-8b6a-fe062fa8b253.png)

UPDATE:
Taking a closer look, we add `avatar` prop for `ChatBubble` and `icon` prop for `ChatAction`; so if we're certain that's always the case (bubbles can have avatar and actions can have icon) I think this is a good separation and you can disregard the comment

Reply:
Yes, we needed actually different components for the structure of the different types of chat items, that's why I separate those.

Reply:
got it, makes sense, thanks!